docs/source/zero-to-binderhub/setup-binderhub.rst

```diff
@@ -110,8 +110,60 @@ where:
 Create ``config.yaml``
 ----------------------
 
-Create a file called ``config.yaml`` and choose the following directions based
-on the registry you are using.
+Create a file called ``config.yaml``.
+
+Exposing JupyterHub
```

Comment:
Rather than repeating yourself I think we could just say `JupyterHub and BinderHub are exposed as LoadBalancers by default.....`, then give both configs and only talk about Nginx ingress once.